logout_page.php

```diff
@@ -32,6 +32,8 @@
 require_api( 'config_api.php' );
 require_api( 'print_api.php' );
 
+$t_logout_redirect = auth_logout_redirect_page();
```

Comment:
I would add a comment here explaining the reason for capturing the value in a temporary value, so it is clear and not optimized away later by mistake.